ARCs/arc-0015.md

```diff
@@ -0,0 +1,254 @@
+---
+arc: 15
+title: Algorand Standard for Encrypted Short Messages 
+description: A standard scheme for encryption/decryption that allows for private messages.
+author: Stéphane Barroso (@sudoweezy), (@urtho)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/*
+status: Draft
+type: Standards Track
+category: ARC
+created: 2021-08-07
+---
+
+# Algorand Standard for Encrypted Short Messages 
+
+## Abstract
+
+Algorand public account addresses (Ed25519) can be converted to DH keys (Curve25519) suitable for encryption.  
+A sender can use a converted public key to encrypt message contents. Transaction signature verification assures the message's authenticity. 
+The recipient can decrypt the message successfully with a derivative of the Algorand private key associated with the public key used for encryption. 
+
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+
+For the rest of the document, we will use the term *Receiver* to represent the set of accounts (public account ed25519, multisig account, contract account, application account)
+
+> Comments like this are non-normative.
+
+
+### Account's message Application
+
+To receive a message, the *Receiver* **MUST** create an application that follows the following convention:
+
+- A Local State named `public_key` **MUST** containing a public account `Ed25519-Address`
```

Comment:
It was part of the original proposal, I kept it because of wallets:
- Already have a way to import Private keys 
- Creating a new ED25519 is part of the algo SDK

But yes, in theory, with this approach, we can ask users to use Curve25519 directly